src/test/java/net/conjur/api/ConjurTest.java

```diff
@@ -23,7 +23,7 @@
  */
 public class ConjurTest {
 
-    private static final String VARIABLE_KEY = "test/testVariable";
+    private static final String VARIABLE_KEY = "testVariable";
```

Comment:
With your permission I have changed the toJson() and toBase64() methods from public to private.
They should not be exposed outside the class.